docs/core/testing/index.md

```diff
@@ -1,75 +1,66 @@
 
-You can also choose between several unit test frameworks:
+## Testing tools
 
-- [xUnit](https://xunit.net/)
-- [NUnit](https://nunit.org)
-- [MSTest](https://github.com/Microsoft/testfx-docs)
+.NET is a multi-language ecosystem, and you can write various test types for [C#](../../csharp/index.yml), [F#](../../fsharp/index.yml), and [Visual Basic](../../visual-basic/index.yml). For each of these languages, you can choose between several test frameworks.
 
-You can learn more in the following walkthroughs:
+### xUnit
 
-:::zone pivot="mstest"
+[xUnit](https://xunit.net) is a free, open source, community-focused unit testing tool for the .NET. Written by the original inventor of NUnit v2, xUnit.net is the latest technology for unit testing C#, F#, VB.NET, and other .NET languages. xUnit.net works with ReSharper, CodeRush, TestDriven.NET, and [Xamarin](/apps/xamarin). It is part of the [.NET Foundation](https://dotnetfoundation.org), and operates under their code of conduct.
 
-- Create unit tests using [*MSTest* and *C#* with the .NET Core CLI](unit-testing-with-mstest.md).
-- Create unit tests using [*MSTest* and *F#* with the .NET Core CLI](unit-testing-fsharp-with-mstest.md).
-- Create unit tests using [*MSTest* and *Visual Basic* with the .NET Core CLI](unit-testing-visual-basic-with-mstest.md).
+You can create tests with xUnit using the .NET in [C#](unit-testing-with-dotnet-test.md), [F#](unit-testing-fsharp-with-dotnet-test.md), and [Visual Basic](unit-testing-visual-basic-with-dotnet-test.md).
 
-:::zone-end
-:::zone pivot="xunit"
+### NUnit
 
-- Create unit tests using [*xUnit* and *C#* with the .NET Core CLI](unit-testing-with-dotnet-test.md).
-- Create unit tests using [*xUnit* and *F#* with the .NET Core CLI](unit-testing-fsharp-with-dotnet-test.md).
-- Create unit tests using [*xUnit* and *Visual Basic* with the .NET Core CLI](unit-testing-visual-basic-with-dotnet-test.md).
+[NUnit](https://nunit.org) is a unit-testing framework for all .NET languages. Initially ported from JUnit, the current production release has been rewritten with many new features and support for a wide range of .NET platforms. It is part of the [.NET Foundation](https://dotnetfoundation.org).
 
-:::zone-end
-:::zone pivot="nunit"
+You can create tests with NUnit using the .NET in [C#](unit-testing-with-nunit.md), [F#](unit-testing-fsharp-with-nunit.md), and [Visual Basic](unit-testing-visual-basic-with-nunit.md).
```

Comment:
Same issue as noted earlier, this text doesn't make clear that the language links are in fact unit testing docs.